src/tests/system/tests/test_sss_override.py

```diff
@@ -108,31 +140,29 @@ def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider:
 
     for i in ["group1", "o-group1"]:
         result = client.tools.getent.group(i)
-        assert result is not None, "Group not found!"
-        assert result.gid == 999999, "Group gid does not match original value!"
-        assert "o-user1" in result.members, "Local override username not found in group!"
+        assert result is not None, f"Group {i} not found!"
+        assert result.gid == 999999, f"Group gid {result.gid} does not match original value!"
+        assert "o-user1" in result.members, "Local override user 'o-user1' not found in group!"
 
     client.sss_override.group("group1").add(name="o-group1", gid=888888)
-
     for i in ["group1", "o-group1"]:
         result = client.tools.getent.group(i)
-        assert result is not None, "Group not found!"
-        assert result.gid == 888888, "Group gid does not match override value!"
-        assert "o-user1" in result.members, "Local override username not found in group!"
+        assert result is not None, f"Group not {i} found!"
+        assert result.gid == 888888, f"Group gid {result.gid} does not match override value!"
+        assert "o-user1" in result.members, "Local override username 'o-user1' not found in group!"
 
 
 @pytest.mark.importance("high")
-@pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.topology(KnownTopology.AD)
-@pytest.mark.topology(KnownTopology.Samba)
-def test_sss_overrides__root_uid_gid_cannot_be_used(client: Client, provider: GenericProvider):
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+def test_sss_override__root_account_cannot_be_used(client: Client, provider: GenericProvider):
     """
     :title: Root user UID/GID cannot be overridden
```

Comment:
It should be `Root user or account with  uid/gid 0 cannot be used` or of similar meaning.

Reply:
ok